crane-cloud/backend
# Added feater for credits expiration with celery

# Description

Added feature for expiration of credits using celery

## Type of change

Please delete options that are not relevant.


## Trello Ticket ID

https://trello.com/c/pTc0nnpQ/1139-add-functionality-for-expiry-of-promotional-credits-expire-after-6-months

## How Can This Be Tested?

Start the application. Run celery worker and beat as specified in the readme. A users credits should expire at midnight if their credits expiration is on that date.  If needed, under tasks/__init__.py change the time of the celery scheduler (beat) on line 66 to test and check for expired credits in a shorter period of time i.e. crontab().


## Checklist:
